src/GraphQLParser.sln

```diff
@@ -15,9 +15,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		..\.gitignore = ..\.gitignore
 		Directory.Build.props = Directory.Build.props
 		Directory.Build.targets = Directory.Build.targets
+		graphql.snk = graphql.snk
 		..\LICENSE.md = ..\LICENSE.md
 		..\assets\logo.64x64.png = ..\assets\logo.64x64.png
+		public.snk = public.snk
```

Comment:
I think this file is not nessessary at all.

Reply:
The file is just included as reference since it is this public key that is used for friend assembly references. 

It is created from the private snk `graphql.snk` using `sn.exe` 

Let me know if I should delete it.

Reply:
Deleted in #265

Reply:
I have no much experience in cryptography, but I suspected that a public key could be obtained from a private key with some standard utility.